src/GuppyDataExplorer/GuppyDataExplorer.jsx

```diff
@@ -13,29 +13,98 @@ import {
   ChartConfigType,
 } from './configTypeDef';
 import './GuppyDataExplorer.css';
+import isEnabled from '../helpers/featureFlags';
 
 class GuppyDataExplorer extends React.Component {
   constructor(props) {
     super(props);
+    let initialState = {};
+    let initialFilter = {};
+
+    // aborting URL decode
+    if (isEnabled('storeExplorerStateInURL')) {
```

Comment:
let's call this feature flag `explorerStoreFilterInURL` to indicate this is a data explorer feature (since it starts with `expolrer...`)